docs/ref/forms/api.txt

```diff
@@ -1464,26 +1473,39 @@ Methods of ``BoundField``
         >>> print(bound_form["subject"].value())
         hi
 
+.. _custom-boundfield:
+
 Customizing ``BoundField``
 ==========================
 
-If you need to access some additional information about a form field in a
-template and using a subclass of :class:`~django.forms.Field` isn't
-sufficient, consider also customizing :class:`~django.forms.BoundField`.
+.. attribute:: Form.bound_field_class
+
+.. versionadded:: 5.2
```

Comment:
I'm not 100% happy with this location for the `.. attribute`, but I do not have a better idea.

Reply:
I have made tweaks to (hopefully) improve the flow of the section and to be better aligned with the exiting sections.